.buildkite/docker-pipeline.yml

```diff
@@ -0,0 +1,20 @@
+steps:
```

Comment:
nit: mind renaming file to `image-release-pipeline.yml` or something else to indicate what it's for